docs/OpenTypeFeatureFileSpecification.md

```diff
@@ -2460,7 +2464,7 @@ context, you need to put the mark and kern rules in different lookups.
 mark sukun <anchor 0 0> @TOP_CLASS;
 
 lookup MARK_POS {
-    position base lam_meem_jeem' <anchor 625 1800> mark @TOP_CLASS alef;
+    position lam_meem_jeem' <anchor 625 1800> mark @TOP_CLASS alef;
```

Comment:
Can you please restore `base` in this line?

Reply:
Sorry, I took this file from the master branch and didn't notice the difference.